### ISSUE TITLE
Friend Tags / User Tags

### ISSUE THREAD

**Issue by stellar-4242** [Vencord/plugin-requests#761]
### Plugin Description

![68747470733a2f2f6e79782e6865702e67672f6e75396b7a30673155](https://github.com/Vencord/plugin-requests/assets/11370958/eb9264ad-022c-49a9-9228-fac387b256d0)

I'm willing to commission an extremely useful plugin that was previously on BetterDiscord, this plugin is extremely useful for users who use Discord as a social platform for networking, I'm constantly losing GFX artists and other like minded individuals due to them changing discord username, closing dms, etc. If I was able to give a user a "tag" and search for it with the quick indexer it would be extremely helpful!

TLDR; 
User Indexer (Via Tags)

Contact me here: roblox.studio.demon

### Existing Plugin for other mod

https://github.com/SrS2225a/BetterDiscord/tree/master/plugins/UserTags

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Scyye** [comment]
https://discord.com/channels/1015060230222131221/1032200195582197831/1253991830501068821
